### PR TITLE
Add connection to API

### PR DESCRIPTION
**Pull request by gustavodiel**
Hello everyone

In this PR, we are going to add a basic connection to the API server.
The API Server's URL are stored in our Environment.

### CODE REVIEW

File: helpers/api_handler.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    Returns URL for all products
+    :return: API's URL for all products
+    """
+    return server_name() + 'products'
+
+
+# Fetchers
+def fetch_all_products():
+    request = requests.get(products_url())
+    print(request.url)
```
Comment (izn) [inline helpers/api_handler.py]: Debugger print missed here?

**Review by izn — APPROVED** [review]
Amazing job :heart:

**Comment by izn** [comment]
[[Integração] Listagem de Produtos](https://trello.com/c/63E7iSzN/2-integra%C3%A7%C3%A3o-listagem-de-produtos)

File: helpers/api_handler.py (hunk 1 above)
Comment (gustavodiel) [reply helpers/api_handler.py]: Oopsy Daisy :grimacing:

**Pull request merged**